src/registrar/models/utility/domain_helper.py

```diff
@@ -33,11 +33,12 @@ def validate(cls, domain: str, blank_ok=False) -> str:
         # Split into pieces for the linter
         domain = cls._validate_domain_string(domain, blank_ok)
 
-        try:
-            if not check_domain_available(domain):
-                raise errors.DomainUnavailableError()
-        except RegistryError as err:
-            raise errors.RegistrySystemError() from err
+        if domain != "":
```

Comment:
changing it to if not blank_ok: will then block the availability check from every running on alternative domain names. On an alternative domain check, blank ok is True. This means "not blank_ok" would then be false for EVERY alternative domain name and we would never check the alternative domain's availability. We don't want that. Also if an alternative domain is blank it just returns "", but if a non alternative domain is blank it will throw an error. Hence, doing !="" both makes sure this can run for alternative domains but also ensures it doesn't check blanks where it doesn't have to